README.md

```diff
@@ -89,6 +89,17 @@ Your `~/.vimrc.local` might look like this:
     highlight NonText guibg=#060606
     highlight Folded  guibg=#0A0A0A guifg=#9090D0
 
+If you don't wish to install a vim plugin from the default set of vim plugins in
+`.vimrc.bundles`, you can ignore the plugin by calling it out with `UnPlug` in
+your `~/.vimrc.bundles.local`.
+blacklist in `~/.vimrc.ignorebundles`:
```

Comment:
Since we're relying on `vim-plug`'s internals, I think we can safely remove this file from the documentation.